packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -475,6 +480,29 @@ async function handler(
     ) {
       throw new HttpError({ statusCode: 403, message: ErrorCode.CancelledBookingsCannotBeRescheduled });
     }
+    const userReschedulingIsOwner = isUserReschedulingOwner(userId, originalRescheduledBooking?.user?.id);
+    if (userReschedulingIsOwner) {
```

Comment:
A team owner/admin can also reschedule a team booking, whether or not they are part of it

Reply:
Fixed.